src/cloudant/design_document.py

```diff
@@ -498,7 +498,12 @@ def fetch(self):
                         view_def.pop('reduce', None),
                         **view_def
                     )
-
+        # dbcopy is transformed to a new storage location, we need to de-transform
+        if 'options' in self:
+            if 'epi' in self['options']:
+                if 'dbcopy' in self['options']['epi']:
+                    for view in self['options']['epi']['dbcopy'].keys():
+                        self['views'][view]['dbcopy'] = self['options']['epi']['dbcopy'][view]
```

Comment:
I wonder if its worth making it some kind of option to do the translation, once DBNext is everywhere newer users using DBCopy may get confused with its location.